internal/stores/sql.go

```diff
@@ -68,6 +68,14 @@ func NewSQLStore(conn gorm.Dialector, migrate bool, persistInterval time.Duratio
 	if err != nil {
 		return nil, modules.ConsensusChangeID{}, err
 	}
+
+	// Disable connection pool.
+	sql, err := db.DB()
+	if err != nil {
+		return nil, modules.ConsensusChangeID{}, err
+	}
+	sql.SetMaxOpenConns(1)
```

Comment:
I wasn't really sneaking it in though it's mentioned in the PR description. But it's a fair point I'm going to remove it from this PR. I just ran into the annoying NDF and I remember us saying that we would hold off on disabling the connection pool because we use a `WAL` and made some query performance improvements. I think we'll have to address the `database is locked` issues soon.